tests/integ/test_function.py

```diff
@@ -1742,6 +1743,10 @@ def test_array_negative(session):
         df.select(array_size([1])).collect()
     assert "'ARRAY_SIZE' expected Column or str, got: <class 'list'>" in str(ex_info)
 
+    with pytest.raises(TypeError) as ex_info:
```

Comment:
use instead `pytest.raises(TypeError, matches='...')`. You do not need the assert then, much easier to read and maintain.

Reply:
I can update them.